frontend/wallet/src/CallTable.re

```diff
@@ -0,0 +1,42 @@
+open Tc;
+
+module Ident = {
+  type t = int;
+};
+
+// TODO: GADT this to support responses tthat aren't just unit
+// call number -> Task completer
+type t = {
+  table: Js.Dict.t(unit => unit),
+  ident: ref(Ident.t),
+};
+
+module Pending = {
+  type t('x) = {
+    ident: Ident.t,
+    task: Task.t('x, unit),
+  };
+};
+
+let make = () => {table: Js.Dict.empty(), ident: ref(0)};
+
+let nextPending = t => {
```

Comment:
How do you feel about taking a src loc and optionally a name? Anything to tie these meaningless sequence numbers to their semantic cause.

Reply:
good idea